playwright/async_api/_generated.py

```diff
@@ -514,16 +514,16 @@ async def fulfill(
 
         Parameters
         ----------
-        status : Union[int, NoneType]
+        status : Optional[int]
             Response status code, defaults to `200`.
-        headers : Union[Dict[str, str], NoneType]
+        headers : Optional[Dict[str, str]]
             Response headers. Header values will be converted to a string.
-        body : Union[bytes, str, NoneType]
+        body : Optional[bytes, str]
             Response body.
-        path : Union[pathlib.Path, str, NoneType]
+        path : Optional[pathlib.Path, str]
```

Comment:
I don't think Optional[X, Y] is a thing.

Reply:
Ya, missed that.